tests/viet_kwik_e_mart/__init__.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+"""This module contains the Kwik-E-Mart MindMeld demo application implemented in Vietnamese"""
+from mindmeld import Application
+
+app = Application(__name__)
+
+__all__ = ["app"]
+
+
+@app.handle(intent="greet")
+def welcome(request, responder):
+    try:
+        responder.slots["name"] = request.context["name"]
+        prefix = "Chào {name}. "
+    except KeyError:
+        prefix = "Xin chào. "
+    responder.reply(prefix)
+
+    statement = "Tôi có thể giúp bạn tìm giờ cửa hàng "
+    "cho Kwik-E-Mart tại địa phương của bạn. "
```

Comment:
This is meant to be a continuation of the previous line, right? Will need parens/brackets.

Reply:
wow good eyes

Reply:
Just making sure if I'm understanding correctly, by needing parens/brackets, we need to rewrite `statement`:

`statement = [
        "Tôi có thể giúp bạn tìm giờ cửa hàng "
        "cho Kwik-E-Mart tại địa phương của bạn. "
]`

________

@serapio That's what you meant by needing parens/brackets?

Reply:
Nevermind, resolved!

Reply:
I would rather have the brackets than the backslash, since it's more visible and helps maintain the indentation.